content/en/apps/concepts/forms.md

```diff
@@ -5,6 +5,11 @@ weight: 3
 description: >
   Building block for all CHT apps
 keywords: app-forms contact-forms collect-forms json-forms
+relatedContent: >
+  apps/guides/forms
+  apps/reference/forms
+  apps/reference/app-settings/forms
+  apps/concepts/care-guides
```

Comment:
Having care-guides first may help to have more general info first, then get into details.

Reply:
updated and ready for re-review